IS7024/Pages/HollywoodBowl.cshtml.cs

```diff
@@ -30,33 +30,28 @@ public void OnGet()
                 string itstime = "~" + realtime + ":" + vars[1] + " PM";
                 ViewData["sun"] = itstime;
 
-
                 string jsonString = webClient.DownloadString("https://api.songkick.com/api/3.0/venues/7211/calendar.json?apikey=Y77nbW56nkfpIpw9");
-                JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("SongkickSchema.json"));
+                JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("SongkickSchemaAll.json"));
```

Comment:
Changed the Schema to SchemaAll which took out the list of acceptable options per variable so that the same schema could be used for all venues.